x-pack/plugins/cloud_security_posture/public/common/constants.ts

```diff
@@ -6,8 +6,58 @@
  */
 
 import { euiPaletteForStatus } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { CLOUDBEAT_EKS, CLOUDBEAT_VANILLA } from '../../common/constants';
 
 const [success, warning, danger] = euiPaletteForStatus(3);
 
 export const statusColors = { success, warning, danger };
 export const CSP_MOMENT_FORMAT = 'MMMM D, YYYY @ HH:mm:ss.SSS';
+
+type PolicyTemplate = 'kspm';
+
+export type CloudPostureIntegrations = Record<
```

Comment:
maybe should just be 
`export type CloudPostureIntegrations = typeof cloudPostureIntegrations`
wdyt?